napari/layers/labels/_tests/test_labels.py

```diff
@@ -1312,7 +1312,7 @@ def test_cursor_ray_3d_rolled():
     labels = Labels(data, scale=(1, 2, 1, 1), translate=(5, 5, 5, 0))
 
     # set the slice to one with data and the view to 3D
-    labels._slice_dims([0, 0, 0, 1], ndisplay=3)
+    labels._slice_dims([0, 0, 0, 1], order=[3, 0, 1, 2], ndisplay=3)
```

Comment:
I still need to work out why these additions are needed and why they weren't needed before. This call to `_slice_dims` actually hits the early return in `_set_view_slice`, so `_slice_input` is updated but no actual slicing happens.